sample-code/src/langchain-openai.ts

```diff
@@ -0,0 +1,27 @@
+import { HumanMessage } from '@langchain/core/messages';
+import { OpenAiChatClient, OpenAiEmbeddingClient } from '@sap-ai-sdk/langchain';
+
+/**
+ * Ask GPT about the capital of France.
+ * @returns The answer from GPT.
+ */
+export async function generate(): Promise<string> {
+  const client = new OpenAiChatClient({ modelName: 'gpt-35-turbo' });
+  const response = await client.generate([
+    [new HumanMessage('What is the capital of France?')]
+  ]);
+  return response.generations[0][0].text;
```

Comment:
Instead of `response.generations[0][0].text;` I'd rather demo some actual langchain usage:

```suggestion
import { StringOutputParser } from "@langchain/core/output_parsers";

const parser = new StringOutputParser();
client.pipe(parser);
```

otherwise, the above code arguably has not much if any benefit or other distinction compared to the standalone OpenAI client